alacritty/build.rs

```diff
@@ -20,7 +20,7 @@ fn main() {
         .unwrap();
 
     #[cfg(windows)]
-    embed_resource::compile("./windows/windows.rc");
+    embed_resource::compile("./windows/alacritty-icon.rc");
```

Comment:
Why did you change that here?

Reply:
The build fails since window-sys creates some lib with `windows` in it's name. So we should have a different name here.

Reply:
Then I'd suggest `alacritty`?

Reply:
I think it'll conflict with the lib from the binary, but we can try...

Reply:
Seems like it's working.